servicex/resources/jwt/user_registration.py

```diff
@@ -53,6 +56,11 @@ def post(self):
 
         try:
             new_user.save_to_db()
+            webhook_url = os.environ.get("SLACK_WEBHOOK_URL", default=None).strip("'")
+            if webhook_url is not None:
```

Comment:
Stylistically better as `if webhook_url:`

Reply:
That is interesting - flake8 complains - at least, that is why I now use the above form.